homeassistant/components/zha/core/device.py

```diff
@@ -474,7 +474,9 @@ def zha_device_info(self):
         device_info["entities"] = [
             {
                 "entity_id": entity_ref.reference_id,
-                ATTR_NAME: entity_ref.device_info[ATTR_NAME],
+                ATTR_NAME: entity_ref.device_info[ATTR_NAME]
```

Comment:
@dmulcahey @adminiuga 

Device trackers no longer create devices. This was causing an issue in the ZHA device tracker tests that I solved like this. Is that ok? 

I also have 2 failing discovery tests.

```
device = {'device_no': 87, 'endpoints': {1: {'device_type': 32768, 'endpoint_id': 1, 'input_clusters': [0, 1, 3, 15, 32], 'outp...nnels': ['basic'], 'entity_class': 'LQISensor', 'entity_id': 'sensor.smartthings_tagv4_77665544_basic_lqi'}, ...}, ...}

device = {'device_no': 87, 'endpoints': {1: {'device_type': 32768, 'endpoint_id': 1, 'input_clusters': [0, 1, 3, 15, 32], 'outp...nnels': ['basic'], 'entity_class': 'LQISensor', 'entity_id': 'sensor.smartthings_tagv4_77665544_basic_lqi'}, ...}, ...}
```

And I have no idea how to fix it.

Reply:
Fixed it by opting ZHA out of this logic. It is only for IP based devices.